ensemble_md/cli/run_REXEE.py

```diff
@@ -322,16 +332,20 @@ def main():
             if (i + 1) % REXEE.n_ckpt == 0:
                 if len(REXEE.g_vecs) != 0:
                     # Save g_vec as a function of time if weight combination was used.
-                    np.save('g_vecs.npy', REXEE.g_vecs)
+                    np.save(args.g_vecs, REXEE.g_vecs)
```

Comment:
So the input argument `g_vecs` was intended to be for specifying an input file like `g_vecs.npy` for extending the simulation (same for the `rep_trajs`), but I do think it makes sense to have them specify the output files to be saved as well. I'll tweak the help message of the argument parser.

Reply:
Hmm actually I think the current help message is fine.